packages/wallet/src/views/Home.jsx

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright (c) Kernel
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { ethers } from 'ethers'
+import { useEffect, useState } from 'react'
+import { linesVector } from '@kernel/common'
+
+import Page from 'components/Page'
+
+const WALLET_STORE_VERSION = '1'
+
+const getItem = (k) => JSON.parse(localStorage.getItem(k))
+
+const loadWallet = () => {
+  const wallet = getItem('wallet')
+  if (wallet.version !== WALLET_STORE_VERSION) {
+    throw new Error('unsupported wallet')
+  }
+  return wallet
+}
+
+const everyoneCards = [
+  {
+    title: 'Claim',
+    description: 'Get set up with some test ETH',
+    url: '/claim',
+    active: false
+  },
+  {
+    title: 'Transact',
+    description: 'Understand how transactions work',
+    url: '/transact',
+    active: false
+  },
+  {
+    title: 'Deploy',
+    description: 'Create your own token now',
+    url: '/deploy',
+    active: false
+  },
+  {
+    title: 'Mint',
+    description: 'Create an NFT in one go',
+    url: '/mint',
+    active: false
+  },
+  {
+    title: 'Contact',
+    description: 'Keep your friends\' addresses safe',
+    url: '/contact',
+    active: false
+  },
+  {
+    title: 'Contribute',
+    description: 'Learn how to improve this with us',
+    url: '/contribute',
+    active: false
+  },
+  {
+    title: 'Interact',
+    description: 'Explore all the best contracts',
+    url: '/interact',
+    active: false
+  },
+  {
+    title: 'Develop',
+    description: 'Write your own code!',
+    url: '/develop',
+    active: false
+  }
+]
+
+const kernelCards = [
```

Comment:
We should keep in mind this is not specific to the Kernel. The concepts we use is different roles represented by a single number visualized by concentric circles.

What you are trying to express in this model is showing different links based on a user's role, but there is no concept of the Kernel in the code base.

The intended way to use this is there's some threshold where you graduate to an outer circle represented as a decrease in the number that is your role.

We are still missing the `application` and `review` apps which will be the primary way to move `members` from the permissionless, open Web3 role to a member role.

Therefore, this should be renamed as `memberCards`.